### DISCUSSION TITLE
macOS supported versions

### DISCUSSION THREAD

**Discussion by chrowe** [monocasual/giada#612] [Q&A]
Is there anywhere that documents what versions of different OSs are supported by the current or previous versions of Giada?

**Comment by chrowe** [comment]
I guess https://github.com/monocasual/giada/blob/master/.github/workflows/packaging.yml#L113 says `runs-on: macos-11` and I can look at the history/blame to see previous versions but was hoping there might be an easier way.

Also, that looks like it was defined relatively recently 
https://github.com/monocasual/giada/commit/86fb93cd76b7445fcf5b231e28ca23ea778ada4e#diff-f52b97956fd8b670a9244eccfef4dbc3406b9fc9c80409edbf176332829a2c98R112 e.g. I am running 10.12.6 which is older than what it defines there.

**Comment by gvnnz** [comment]
Hi @chrowe , sorry for the delay. 

> Is there anywhere that documents what versions of different OSs are supported by the current or previous versions of Giada?

Yes, we have it listed here: https://www.giadamusic.com/documentation-introduction. But that info might be out of date. What macOS version are you using at the moment? Does Giada 0.23.0 works fine on it (once the quarantine/Gatekeeper issues have been fixed)?
